src/content/fr/2019/css.md

```diff
@@ -0,0 +1,735 @@
+
+{{ figure_markup(
+  image="fig8.png",
+  caption="Popularité des types de sélecteurs, par sélecteur.",
+  description="Diagramme à barres montrant que 94&nbsp;% des sélecteurs incluent le sélecteur de classe pour ordinateur de bureau et mobile, tandis que 7&nbsp;% des sélecteurs de bureau incluent le sélecteur par ID (8&nbsp;% pour mobile).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=351006989&format=interactive"
+  )
+}}
+
+### Sélecteurs par attributs
+
+CSS a des sélecteurs de comparaison très puissants. Ce sont des sélecteurs comme `[target="_blank"]`, `[attribute^="value"]`, `[title~="rad"]`, `[attribute$="-rad"]` ou `[attribut*="valeur"]`. Les utilisez-vous&nbsp;? Vous pensez qu’ils sont beaucoup utilisés&nbsp;? Comparons comment ceux-ci sont utilisés avec les ID et les classes sur le Web.
+
+{{ figure_markup(
+  image="fig9.png",
+  caption="Popularité des opérateurs par sélecteur d’attribut ID.",
+  description="Diagramme à barres montrant la popularité des opérateurs utilisés par les sélecteurs d’attributs d’ID. Environ 4&nbsp;% des pages de bureau et mobiles utilisent «`*=`» et «`^=`». 1&nbsp;% des pages utilisent «`=`» et «`$=`». 0&nbsp;% utilise «`~=`».", 
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=695879874&format=interactive"
+  )
+}}
+
+{{ figure_markup(
+  image="fig10.png",
+  caption="Popularité des opérateurs par sélecteur d’attribut de classe.",
+  description="Diagramme à barres montrant la popularité des opérateurs utilisés par les sélecteurs d’attributs de classe. 57&nbsp;% des pages utilisent des «`*=`». 36&nbsp;% utilisent «`^=`». 1&nbsp;% utilise «`=`» et «`$=`». 0&nbsp;% utilise «`~=`».",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQO5CabwLwQ5Lj1_9bbEFnFM1qEqCorymaBHrcaNiMSJ7sYDKHUI5iish5VAS-SxN447UTW-1-5-OjE/pubchart?oid=377805296&format=interactive"
+  )
+}}
+
+Ces opérateurs sont beaucoup plus populaires avec les sélecteurs de classe que les identifiants, ce qui semble naturel puisqu’une feuille de style a généralement moins de sélecteurs d’identifiants que de sélecteurs de classe, mais il reste agréable de voir les utilisations de toutes ces combinaisons.
+
+### Classes par élément
+
+Avec la montée en puissance des approches CSS comme OOCSS, <i lang="en">Atomic CSS</i> et <i lang="en">functional CSS</i> qui peuvent utiliser 10 classes ou plus sur un élément pour créer son design, peut-être verrions-nous des tendances intéressantes. La requête une fois revenue est malheureusement assez peu intéressante, la médiane sur mobile et ordinateur de bureau étant d’une classe par élément.
+
+{{ figure_markup(
+  caption="Le nombre médian de noms de classes par attribut class (bureau et mobile).",
+  content="1",
+  classes="big-number"
+)
+}}
+
+## Disposition
+
+### Flexbox
```

Comment:
Either

```suggestion
### <i lang="en">Flexbox</i> {flexbox}
```

or

```suggestion
### `flexbox` {flexbox}
```

Reply:
And same below. I prefer the use of a code wrapping myself, but an `lang` wrapping is ok too.

Reply:
And same for Grid below too.